README.md

```diff
@@ -1,12 +1,45 @@
-![CI](https://github.com/nearform/bench-template/actions/workflows/ci.yml/badge.svg?event=push)
+![CI](https://github.com/nearform/fastify-constraints/actions/workflows/ci.yml/badge.svg?event=push)
 
-# Bench Template
+Fastify plugin to add constraints to multiple routes
```

Comment:
include a h1 with the name of the plugin, before the description, e.g. # fastify-constraints